engine/apps/api/views/alert_receive_channel.py

```diff
@@ -90,6 +78,22 @@ class AlertReceiveChannelView(
 
     filterset_class = AlertReceiveChannelFilter
 
+    rbac_permissions = {
+        "list": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_READ],
+        "retrieve": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_READ],
+        "integration_options": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_READ],
+        "counters": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_READ],
+        "counters_per_integration": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_READ],
+        "create": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_WRITE],
+        "update": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_WRITE],
+        "partial_update": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_WRITE],
+        "destroy": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_WRITE],
+        "send_demo_alert": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_WRITE],
+        "change_team": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_WRITE],
+        # TODO: is this the correct permission?
+        "preview_template": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_WRITE],
```

Comment:
previously [this was](https://github.com/grafana/oncall/blob/dev/engine/apps/api/views/alert_receive_channel.py#L72) `IsAdminOrEditor`, I believe this diff makes sense